### PR TITLE
New package: lsd-0.16.0

### LINKED ISSUE #13968: Package Request : lsd
The next gen ls command
[https://github.com/Peltoche/lsd](url)

### PR DESCRIPTION
**Pull request by lugubris**
fixes #13968

### CODE REVIEW

**Comment by lugubris** [comment]
I hacked the config from alacritty's config so let me know if something is missing or is extra.

File: srcpkgs/lsd/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'lsd'
+pkgname=lsd
+version=0.16.0
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config"
+#makedepends="rust cargo"
```
Comment (Hoshpak) [inline srcpkgs/lsd/template]: commented out lines that are not needed should be removed.

File: srcpkgs/lsd/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+depends="awesome-terminal-fonts"
```
Comment (Hoshpak) [inline srcpkgs/lsd/template]: This package does not exist, maybe you wanted `font-awesome` or `font-awesome5`?

File: srcpkgs/lsd/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+short_desc="The next gen ls command"
+maintainer="Lugubris <lugubris@disroot.org>"
+license="Apache-2.0"
+homepage="https://github.com/Peltoche/${pkgname}"
```
Comment (Hoshpak) [inline srcpkgs/lsd/template]: please don't use variables in the homepage field. This makes it harder to just copy the URL

File: srcpkgs/lsd/template [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+changelog="https://raw.githubusercontent.com/Peltoche/${pkgname}/${version}/CHANGELOG.md"
+distfiles="https://github.com/Peltoche/${pkgname}/archive/${version}.tar.gz"
+checksum=e2406748d78431a1c03bdd2404a204a006c19905d926e41a36587b93a791e003
+
+do_install() {
```
Comment (Hoshpak) [inline srcpkgs/lsd/template]: why is that function necessary? It seems to mostly replicate the `build_style` function.

Comment (lugubris) [reply srcpkgs/lsd/template]: does the template look good now?

**Pull request merged**